src/shell/commands/disk_rebalance.cpp

```diff
@@ -5,45 +5,14 @@
 #include "shell/commands.h"
 #include "shell/argh.h"
 #include "shell/command_output.h"
+#include "shell/command_helper.h"
```

Comment:
could you remove `#include "shell/command_helper.h"`?

Reply:
I think it seem should include `command_utils.h` which contains `validate_cmd` method